library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -195,6 +204,41 @@ public void transformPage(View page, float position) {
         setCurrentDate(currentMonth);
     }
 
+    private void setupChildren() {
+        int tileSize = getResources().getDimensionPixelSize(R.dimen.mcv_default_tile_size);
+
+        root = new LinearLayout(getContext());
+        root.setOrientation(LinearLayout.VERTICAL);
+        root.setWeightSum(8);
+        root.setClipChildren(false);
+        root.setClipToPadding(false);
+        LayoutParams p = new LayoutParams(tileSize * 7, tileSize * 8);
```

Comment:
should you throw the multipliers into a constant? I see you're reusing them in a few other spots